<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1051</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1051</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Datos del Curso:</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtIdCarrera">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNombre">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::VisualMoveStyle</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Créditos:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCreditos">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDuracion">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>80</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Duración:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modalidad:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboModalidad">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>40</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Opciones:</string>
    </property>
    <widget class="QPushButton" name="btnRegistrar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="text">
      <string>Registrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnActualizar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="text">
      <string>Actualizar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLimpiar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="text">
      <string>Limpiar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>1031</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Registro de Cursos:</string>
    </property>
    <widget class="QTableWidget" name="tblCarrera">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>31</y>
       <width>1001</width>
       <height>551</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>601</width>
       <height>551</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>11111</width>
       <height>11111</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>195</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID_Carrera</string>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom_Carrera</string>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Creditos_Totales</string>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Duración</string>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modalidad</string>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
       </font>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
